storage-blob/runtime/src/main/java/io/quarkiverse/azureservices/azure/storage/blob/runtime/StorageBlobConfiguration.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkiverse.azureservices.azure.storage.blob.runtime;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "azure.storage.blob", phase = ConfigPhase.RUN_TIME)
+public class StorageBlobConfiguration {
+
+    /**
+     * The connection string of Azure Storage Account.
+     */
+    @ConfigItem
+    public String connectionString;
```

Comment:
This really should be 3 separate configuration properties.  Mostly so the account key can be stored in a Vault Secret.

Reply:
Good point. So far I expose `quarkus.azure.storage.blob.connection-string` for the simplicity, will consider more config properties that're supported by `com.azure.storage.blob.BlobServiceClient` in the future, as well as storing account key in a Vault Secret.